bitnami/mongodb/values-production.yaml

```diff
@@ -8,6 +8,12 @@
 #     - myRegistryKeySecretName
 #   storageClass: myStorageClass
 
+## Global namespace override
+## Override the namespace for resource deployed by the chart, but can itself be overridden by the local namespaceOverride
+##
+# global:
```

Comment:
Do not duplicate the `global` section, add the parameter to the already existing one